docs/fitting_types.rst

```diff
@@ -256,6 +256,37 @@ Let's plot the model for some kinetics constants::
    :width: 300px
    :alt: ODE integration
 
+More common examples, such as dampened harmonic oscillators also work as expected:
+
+  # oscillator strength
+  k = symfit.Parameter()
+  # mass, just there for the physics
+  m = symfit.Parameter(fixed=True)
```

Comment:
If mass is not something that was to be fitted but rather a constant, we can just leave it as `m = 1` for example. I think that's clearer.